### ISSUE TITLE
Server

### ISSUE THREAD

**Issue by GFOXSH** [Chocimier/skrypty-obml#1]
Hello! Can I use your scripts to make my own Opera Mini server (for personal use). How I can do it?

**Comment by Chocimier** [comment]
Yes, they could be useful. First, you should find a way to redirect requests from Opera Mini to your server. I can help you with further steps.

**Comment by GFOXSH** [comment]
I can hex-edit libom.so file to change server IP and port. But I do not know what to do next.

**Comment by Chocimier** [comment]
You need to:
1. set up some programatically controlled web browser (maybe headless chrome)
2. configure it to call `zbudujObml` from `kreślidło_obml.js` on page load
3. create server that passes request to browser, and then replies with generated obml

Generating of obml when page is loaded is only task that these scripts do.

**Comment by GFOXSH** [comment]
I'm really don't know, how to do it... Can you make detailed guide for me?

**Comment by Chocimier** [comment]
For a start, I'm sorry for my misleading comments. It is possible to do what you want, but it need some research and experiments. I did not do that before, and don't do that for you.

That said, what did you do up to now and what is now your specific problem? We can switch to private mail if you want, just use address from git log.
